sentinel-demo/sentinel-demo-rocketmq/src/main/java/com/alibaba/csp/sentinel/demo/rocketmq/PullConsumerDemo.java

```diff
@@ -76,14 +76,11 @@ public static void main(String[] args) throws MQClientException {
                     putMessageQueueOffset(mq, nextOffset);
                     consumer.updateConsumeOffset(mq, nextOffset);
                     switch (pullResult.getPullStatus()) {
-                        case FOUND:
-                            break;
-                        case NO_MATCHED_MSG:
-                            break;
                         case NO_NEW_MSG:
                             break SINGLE_MQ;
+                        case FOUND:
```

Comment:
emmm,I think both  the previous code and your code are clear, It’s just that everyone’s code habits are different.

Reply:
modification suggestions from intellij idea code inspection, so just a suggestions

Reply:
It's okay :)